pkg/daemon/ceph/client/image.go

```diff
@@ -84,7 +85,7 @@ func CreateImage(context *clusterd.Context, clusterName, name, poolName, dataPoo
 	}
 
 	buf, err := ExecuteRBDCommandNoFormat(context, clusterName, args)
-	if err != nil {
+	if err != nil && !strings.Contains(string(buf), fmt.Sprintf("%s already exists", name)) {
```

Comment:
it would be nice if we could rely on the exit code from this command instead of parsing the string output.  Could you try this scenario out and take a look at the [ExitStatus() function](https://github.com/rook/rook/blob/release-0.8/pkg/util/exec/error.go#L39) to see if that returns a reliable error code to use instead.  Example usage can be found at https://github.com/rook/rook/blob/release-0.8/pkg/operator/ceph/cluster/osd/remove.go#L51

Reply:
@jbw976 Thanks for the tip. I was initially trying the same. just didn't know how. :)

Reply:
@jbw976 is there ever a scenario where we will have two pvc images with the same UID in the same pool? i feel like some more checks either on the size and/or image name might be good to do prior to linking the pv with the image.

Reply:
I don't **think** there would ever be 2 PVCs with the same UID period, let alone in the same pool.  More than one consumer may try to reference the same PVC, but I would expect that the Kubernetes volume subsystem would not create 2 of them with the same UID in response to that.